browser/widevine/widevine_permission_request.cc

```diff
@@ -18,26 +18,28 @@
 #include "components/prefs/pref_service.h"
 #include "components/url_formatter/elide_url.h"
 #include "components/vector_icons/vector_icons.h"
-#include "content/public/browser/browser_context.h"
 #include "content/public/browser/web_contents.h"
 #include "ui/base/l10n/l10n_util.h"
 
 // static
 bool WidevinePermissionRequest::is_test_ = false;
 
 WidevinePermissionRequest::WidevinePermissionRequest(
-    content::WebContents* web_contents,
+    PrefService* prefs,
+    const GURL& requesting_origin,
     bool for_restart)
     : PermissionRequest(
           std::make_unique<permissions::PermissionRequestData>(
               std::make_unique<permissions::ContentSettingPermissionResolver>(
                   permissions::RequestType::kWidevine),
               false,
-              web_contents->GetVisibleURL()),
```

Comment:
@simonhong We no longer use `GetVisibleURL`, but instead pass in `GetLastCommitedURL`. Is that change OK?

Reply:
Yeah, I think it's ok as here needs the url not for displaying.